### PR TITLE
1110 enhance uiux for cv analyse

### PR DESCRIPTION
**Pull request by baolanlequang**
- [x] rather 1-story 1-commit than sub-atomic commits

- [x] commit title is meaningful =>  git history search

- [x] commit description is helpful => helps the reviewer to understand the changes

- [ ] code is up-to-date with the latest developments of the target branch (rebased to it or whatever) => :fast_forward:-merge for linear history is favoured

- [x] added code is linted

- [ ] tests are passing (at least locally): we still have some random test failure on CI. thinking of asking spec/examples.txt to be commited

- [x] in case the changes are visible to the end-user,  video or screenshots should be added to the PR => helps  with user testing

- [ ] testing coverage improvement is improved.

- [ ] CHANGELOG :  add a bullet point on top (optional: reference to github issue/PR )

- [ ] parallele PR for documentation  on docusaurus  if the feature/fix is tagged for a release

### CODE REVIEW

**Comment by baolanlequang** [comment]
Demo video for new UI/UX

https://user-images.githubusercontent.com/9006482/216930680-27479f5b-31cb-4e26-96dd-74b67147be21.mp4

File: app/packs/src/apps/mydb/elements/details/ViewSpectra.js [hunk 1]
```diff
@@ -598,6 +598,7 @@ class ViewSpectra extends React.Component {
               descriptions={descriptions}
               canChangeDescription
               onDescriptionChanged={this.onSpectraDescriptionChanged}
+              userManualLink={{cv: 'https://www.chemotion.net/chemotionsaurus/docs/eln/chemspectra/cvanalysis'}}
```
Comment (PiTrem) [inline app/packs/src/apps/mydb/elements/details/ViewSpectra.js:601]: path to docs has changed :

https://chemotion.net/docs/chemspectra/cv ?

File: app/packs/src/stores/alt/stores/SpectraStore.js [hunk 2]
```diff
@@ -77,12 +77,16 @@ class SpectraStore {
 
   handleLoadSpectra({ fetchedFiles, spcInfos }) {
     const spcMetas = this.decodeSpectra(fetchedFiles);
```
Comment (PiTrem) [inline app/packs/src/stores/alt/stores/SpectraStore.js:79]: can it happen that spcMetas is undefined or null?

**Review by PiTrem — CHANGES_REQUESTED** [review]
link to docu needs an update

also be sure that spcMetas.map does not break things

**Review by PiTrem — APPROVED** [review]

**Pull request merged**